genre.rb

```diff
@@ -0,0 +1,11 @@
+require './item'
```

Comment:
Please remove or comment out this line as it would throw errors when you run the unit tests